pkg/policy/manager.go

```diff
@@ -282,7 +286,7 @@ func (m *ManagerImpl) runSubPolicyOrQuery(
 				subPolicyMap[k] = v
 			}
 		}
-		return exec.ExecutePolicies(ctx, execReq, subPolicyMap)
+		return exec.ExecutePolicy(ctx, execReq, subPolicyMap[subPath])
```

Comment:
I guess the above stuff isn't needed anymore since you only pass the policy which already contains all its subpolicies?

Reply:
fixed suggestion